wagtail/locales/views.py

```diff
@@ -117,3 +117,11 @@ def get_common_view_kwargs(self, **kwargs):
 
     def get_form_class(self, for_update=False):
         return LocaleForm
+
+    def get_permissions_to_register(self):
+        # Only register these permissions (and not others e.g. "view_locale")
+        return (
+            super()
+            .get_permissions_to_register()
+            .filter(codename__in=["add_locale", "change_locale", "delete_locale"])
+        )
```

Comment:
Rather than overriding this individually for each model where we're not using the view permission, what if we excluded the view permission as standard whenever the inspect view is disabled?

(This would be a change of behaviour for snippets, but I'd be okay with that.)

Reply:
Thanks! Done in 2fc7b4d.